src/overlays/actors/ovl_En_Dha/z_en_dha.c

```diff
@@ -15,7 +15,20 @@ void EnDha_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnDha_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnDha_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void EnDha_SetupAction(EnDha* this, EnDhaActionFunc actionFunc);
+void func_809EC9C8(EnDha* this);
+void func_809ECA50(EnDha* this, GlobalContext* globalCtx);
+void func_809ECF60(EnDha* this);
+void func_809ECF8C(EnDha* this, GlobalContext* globalCtx);
+void EnDha_SetDeadAction(EnDha* this);
+void EnDha_Die(EnDha* this, GlobalContext* globalCtx);
+void EnDha_UpdateHealth(EnDha* this, GlobalContext* globalCtx);
+s32 EnDha_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx);
+void EnDha_OverridePostDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx);
```

Comment:
Avoid declaring functions like SetupAction or callbacks at the top of the actor unless absolutely necessary. As far as I know, these functions are always defined in an order that doesn't require declaring them at a previous point, so we've been omitting them. Some of the other declarations are probably not necessary either but those are fine imo.
```suggestion
void func_809EC9C8(EnDha* this);
void func_809ECA50(EnDha* this, GlobalContext* globalCtx);
void func_809ECF60(EnDha* this);
void func_809ECF8C(EnDha* this, GlobalContext* globalCtx);
void EnDha_SetDeadAction(EnDha* this);
void EnDha_Die(EnDha* this, GlobalContext* globalCtx);
void EnDha_UpdateHealth(EnDha* this, GlobalContext* globalCtx);
```